pkg/virtctl/vm/add_volume_test.go

```diff
@@ -185,3 +159,29 @@ var _ = Describe("Add volume command", func() {
 		Entry("addvolume dv, with LUN-type disk should call VMI endpoint", "addvolume", "testvmi", "testvolume", true, expectVMIEndpointAddVolume, "--disk-type", "lun"),
 	)
 })
+
+func createTestDataVolume() *v1beta1.DataVolume {
+	return &v1beta1.DataVolume{
+		ObjectMeta: k8smetav1.ObjectMeta{
+			Name: "testvolume",
+		},
+	}
+}
+
+func createTestPVC() *k8sv1.PersistentVolumeClaim {
+	return &k8sv1.PersistentVolumeClaim{
+		ObjectMeta: k8smetav1.ObjectMeta{
+			Name: "testvolume",
+		},
+	}
+}
+
+func verifyVolumeSource(volumeOptions interface{}, useDv bool) {
```

Comment:
Can you tell why `volumeOptions` needs to be an interface?

Reply:
I just extracted it to static func, it was like that as well before
but you are correct, i will try to convert it, once it returns from `DoAndReturn` as interface, it can be casted just once instead several times